global-script.lua

```diff
@@ -3982,17 +3990,24 @@ function onPlayerDisconnect(player)
 end
 
 function isPowerCard(card)
-    if card.type == "Card" and (card.hasTag("Minor") or card.hasTag("Major") or card.hasTag("Unique")) then
-        return true
-    end
-    return false
+    return card.type == "Card" and (card.hasTag("Minor") or card.hasTag("Major") or card.hasTag("Unique"))
 end
 function onObjectSpawn(obj)
     if isPowerCard(obj) then
-        applyPowerCardContextMenuItem(obj)
+        applyPowerCardContextMenuItems(obj)
     end
 end
-function applyPowerCardContextMenuItem(card)
+function applyPowerCardContextMenuItems(card)
+    card.addContextMenuItem(
+        "Discard (to 2nd hand)",
+        function(player_color)
+            if not isObjectInHand(card, player_color, 2) then
+                -- `deal` is buggy and seems to have magic logic associated
+                -- with card visibility.
+                card.setPosition(secondHandDiscardPositions[player_color])
```

Comment:
so once this gets swapped over to not used hardcoded magic coordinates (especially since those hand positions can change), you'll probably want to add an offset to the right to always put the card at the end

Reply:
Done.